Assets/AppCenter/link.xml

```diff
@@ -8,6 +8,7 @@
     <!-- Keep all our assemblies. -->
     <assembly fullname="Microsoft.AppCenter" preserve="all" ignoreIfMissing="1"/>
     <assembly fullname="Microsoft.AppCenter.Analytics" preserve="all" ignoreIfMissing="1"/>
+    <assembly fullname="Microsoft.AppCenter.Auth" preserve="all" ignoreIfMissing="1"/>
```

Comment:
@annakocheshkova Do you know if we need this one here?

Reply:
No it's for UWP afair

Reply:
This will be fixed in the next PR.